src/__test__/calculate.test.js

```diff
@@ -0,0 +1,11 @@
+import calculate from '../logic/calculate';
+
+test('test function for key AC', () => {
+  const obj = {
+    total: null,
+    next: null,
+    operation: null,
+  };
+  const buttonName = 'AC';
+  expect(calculate(obj, buttonName)).toEqual({ total: null, next: null, operation: null });
+});
```

Comment:
- [ ] According to the requirements, **all the functionalities & scenarios** of the operate.js and calculate.js modules should be covered using Jest. Kindly ensure that you have tests written to cover all the functionalities of calculate.js